<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task_one</class>
 <widget class="QWidget" name="task_one">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 255, 243);</string>
  </property>
  <widget class="QLabel" name="label_task">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 94, 83);
color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Return to menu</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>281</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>equation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_first">
      <property name="text">
       <string>first</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_second">
      <property name="text">
       <string>second</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_accuracy">
      <property name="text">
       <string>accuracy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_answer">
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
	border: 1px solid rgb(255,94,83);
	border-radius: 550px;
	background-color: rgb(255,255,255);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_send">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 94, 83);
color: rgb(255, 255, 255);
font: 9pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
